<launch>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find slam_project)/launch/multi_slam.rviz" />
  <node name="points_and_lines" pkg="slam_project" type="points_and_lines" output="screen"/>
  <node name="slam_runner_node" pkg="slam_project" type="slam_runner_node" >
    <rosparam command="load" file="$(find slam_project)/config/config.yaml" />
  </node>
  <node name="data_reader_node" pkg="slam_project" type="data_reader_node" />
</launch>
